### DISCUSSION TITLE
FreeSewing v3 breaking changes

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#1684] [Announcements]
See: https://freesewing.dev/guides/v3/migration

**Comment by bobgeorgethe3rd** [comment]
What do I need to do to make sure Holmes is prepped for the depreceation?

**Reply by joostdecock** [reply]
No worries. I've already adapted ito the changes in plugin-gore.

**Reply by bobgeorgethe3rd** [reply]
I see awesome. :) 👍

**Comment by BenJamesBen** [comment]
FYI, the changes I needed to make after updating my workspace to the v3 develop branch in order to run my localhost lab:
```
$ nvm install lts/gallium
$ nvm use lts/gallium
$ nvm alias default lts/gallium
$ # (restarted my other terminal windows so they also could pick up the changes)
$ npm install yarn --global
```

Edit, added: To gain access to the new v2 branch in my local repository (which was cloned from my GitHub fork which did not contain the new v2 branch), I had to: 
```
$ git fetch upstream
```

**Comment by joostdecock** [comment]
See: https://freesewing.dev/guides/v3/migration

**Comment by joostdecock** [comment]
Outdated. Closing.
